ioservice/io_foms.c

```diff
@@ -1534,12 +1534,8 @@ static int zero_copy_initiate(struct m0_fom *fom)
 		used_size = rwfop->crw_desc.id_descs[fom_obj->
 						fcrw_curr_desc_index].bdd_used;
 
-		if (dom->nd_xprt == &m0_net_lnet_xprt) {
-			segs_nr = used_size / max_seg_size;
-		} else {
-			segs_nr = 1;
-			(void)max_seg_size;
-		}
+		segs_nr = ((used_size / max_seg_size) + \
```

Comment:
No need to have the backslash "\".
Please remove it.

Reply:
Removed backslash